package.json

```diff
@@ -21,22 +21,25 @@
     "test:passwords": "PASSWORD_STRESS_TEST=true jest generate.test.js",
     "test:unit": "jest",
     "test:integration": "npm run build && npm run copy-assets && playwright test",
+    "test:integration:fast": "playwright test --timeout 10000 --workers 8",
```

Comment:
A shortcut that skips the build, runs with more workers and a shorter timeout so tests fail faster. This is great for faster test iterations. The other is safer (you're always re-creating the build).